source/manuals/alerting-with-prometheus-for-paas.html.md.erb

```diff
@@ -24,7 +24,7 @@ Use the [Prometheus dashboard][1] to experiment writing your alert as a [PromQL]
 
 Your expression must contain an `org` label, which refers to your PaaS organisation. This makes you sure it only uses metrics from your team. Although you can use the `job` label for this, it is not guaranteed to be unique to your team.
 
-You should only include timeseries for the PaaS space you wish to alert on, for example production. For timeseries produced by application client libraries, include `space="production"`. For timeseries produced by the [paas-metric-exporter][8] include `exported_space="production"`.
+You should only include timeseries for the PaaS space you wish to alert on, for example production.
```

Comment:
I think it would still be worth including the `space="production"` ie.

You should only include timeseries for the PaaS space you wish to alert on, for example only including production using the `space="production"` label.

Reply:
Sounds sensible